src/index.js

```diff
@@ -73,7 +73,7 @@ class HMIApp extends React.Component {
         this.onTouchMove = this.onTouchMove.bind(this);
         this.onTouchEnd = this.onTouchEnd.bind(this);
         this.onTouchEvent = this.onTouchEvent.bind(this);
-        store.dispatch(resetTimeout(props.resetPeriodValue));
+        store.dispatch(resetTimeout({resetPeriod: DEFAULT_RESET_TIMEOUT}));  
```

Comment:
@LiliiaShlikhtLuxoft Why resetTimeout intialize as `true` by default, then it's dispatched here in this place? Why the value couldn't be assigned while initializing? Looks weird

Reply:
@ValeriiMalkov fixed